### PR TITLE
chore: project addition by [palakkgoyal] #1754

### PR DESCRIPTION
**Pull request by Palakkgoyal**
## Related Issue

This PR closes issue no. #1754 (project addition by palakkgoyal)

## Description

I added a new project in which techstacks used include React and Tailwind.

## Screenshots

Screenshot 2023-07-11 000253
Screenshot 2023-07-11 000355
Screenshot 2023-07-11 000414
Screenshot 2023-07-11 000521

### CODE REVIEW

File: DB/projects.json [hunk 1]
```diff
@@ -86,6 +86,22 @@
...
+    "gitHub": "https://github.com/Palakkgoyal",
+    "LinkedIn": "https://www.linkedin.com/in/palak-goyal-037a02269/",
+    "Twitter": "https://twitter.com/Palaktwts"
+  },
+  "Projects": [
+    {
+      "link": "https://github.com/Palakkgoyal/cryptoApp",
+      "title": "Crypto Landing Page",
+      "description": "It is a clone of crypto.com landing page",
+      "tech": ["React", "Tailwind"]
```
Comment (priyankarpal) [inline DB/projects.json:101]: change to lowercase
```suggestion
      "tech": ["reactjs", "tailwindcss"]
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by Palakkgoyal** [comment]
@priyankarpal I did the required changes. Please review the PR.

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Comment by Palakkgoyal** [comment]
Won't you add a gssoc level label to it?

**Comment by priyankarpal** [comment]
> Won't you add a gssoc level label to it?

We are not providing any labels for project additions as they are not associated with solving any issues.

**Pull request merged**